### PR TITLE
{Doc} Added Extension installation section in doc/extensions/README.md

### PR DESCRIPTION
**Pull request by haroldrandom**
Fix https://github.com/Azure/azure-cli/issues/6454

**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: doc/extensions/README.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ What an Extension is not
 - If you're looking to build your own CLI using `az` patterns, see [knack](https://github.com/Microsoft/knack).
 
 
+How to find and install an Extension
+------------------------
+
+- List all available extensions: `az extensino list-available`
```
Comment (fengzhou-msft) [inline doc/extensions/README.md]: typo `extensino`

File: doc/extensions/README.md [hunk 2]
```diff
@@ -25,6 +25,16 @@ What an Extension is not
...
+
+- Install an extension: `az extension install --name <extension-name>`
+
+More details on usage in [Extensions for Azure CLI 2.0](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/docs-ref-conceptual/azure-cli-extensions-overview.md#install-extensions)
```
Comment (fengzhou-msft) [inline doc/extensions/README.md]: Shall we link to live doc: https://docs.microsoft.com/cli/azure/azure-cli-extensions-overview?view=azure-cli-latest#install-extensions

File: doc/extensions/README.md (hunk 1 above)
Comment (haroldrandom) [reply doc/extensions/README.md]: Fixed

File: doc/extensions/README.md (hunk 2 above)
Comment (haroldrandom) [reply doc/extensions/README.md]: Changed

**Pull request merged**